lib/si-layer-cache/src/db.rs

```diff
@@ -116,8 +120,22 @@ where
                 compute_executor.clone(),
                 tracker.clone(),
                 token.clone(),
-                30,
-                30
+                // TODO(nick,jacob): ask Scott about this.
+                25,
+                // TODO(nick,jacob): ask Scott about this.
+                25
+            ),
+            create_layer_cache(
+                change_batch::CACHE_NAME,
+                pg_pool.clone(),
+                cache_config.clone(),
+                compute_executor.clone(),
+                tracker.clone(),
+                token.clone(),
+                // TODO(nick,jacob): ask Scott about this.
+                5,
+                // TODO(nick,jacob): ask Scott about this.
+                5
```

Comment:
@sprutton1 @zacharyhamm we are taking 5% headroom from CAS. All the tunables added up to 100% before and we felt as though this might be the lowest impact location. Is this the right move?
cc @jhelwig

Reply:
We validated this was the right call 👍